From 267516c76208e63b5a3ec7d4dead6c329d8763bb Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Tue, 6 Feb 2024 22:46:59 +0100
Subject: [PATCH] configure.ac: move PY_STDLIB_MOD_SET_NA further up

We will need PY_STDLIB_MOD_SET_NA in next patches further up in the
configure.ac script.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 configure.ac | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/configure.ac
+++ b/configure.ac
@@ -95,6 +95,12 @@ AC_DEFUN([PY_CHECK_EMSCRIPTEN_PORT], [
   AS_VAR_POPDEF([py_libs])
 ])
 
+# stdlib
+AC_DEFUN([PY_STDLIB_MOD_SET_NA], [
+  m4_foreach([mod], [$@], [
+    AS_VAR_SET([py_cv_module_]mod, [n/a])])
+])
+
 AC_SUBST([BASECPPFLAGS])
 if test "$srcdir" != . -a "$srcdir" != "$(pwd)"; then
     # If we're building out-of-tree, we need to make sure the following
@@ -7701,12 +7707,6 @@ fi
 AC_SUBST([_PYTHREAD_NAME_MAXLEN])
 
 
-# stdlib
-AC_DEFUN([PY_STDLIB_MOD_SET_NA], [
-  m4_foreach([mod], [$@], [
-    AS_VAR_SET([py_cv_module_]mod, [n/a])])
-])
-
 # stdlib not available
 dnl Modules that are not available on some platforms
 AS_CASE([$ac_sys_system],
